batch.go

```diff
@@ -16,10 +16,13 @@ type BatchWithFlusher struct {
 var _ dbm.Batch = &BatchWithFlusher{}
 
 // Ethereum has found that commit of 100KB is optimal, ref ethereum/go-ethereum#15115
-// var defaultFlushThreshold = 100000
+var DefaultFlushThreshold = 100000
 
 // NewBatchWithFlusher returns new BatchWithFlusher wrapping the passed in batch
 func NewBatchWithFlusher(db dbm.DB, flushThreshold int) *BatchWithFlusher {
+	if flushThreshold <= 0 {
```

Comment:
should fllushthreshold be a unsigned integer here? not sure when it would be less than 0

Reply:
it's because `batch.GetByteSize()` return `int`, we want `flushThreshold` to also be int so we don't have to do any conversion from `int` to `uint`